lib/package.json

```diff
@@ -12,30 +12,17 @@
     "auth",
     "oauth2"
   ],
-  "exports": {
```

Comment:
Why was this removed?

Reply:
Have you looked into merge this `package.json` with the top-level one?

Reply:
it's handled with the new version of Angular. I double checked the package.json in the dist dir

(there's also a warning in the console if you leave it)

Reply:
sorry, just saw your second comment.

i'm not sure i understand the question. this is the package.json we distribute, so i'd think we wouldn't want to combine it with the root project level one. (i bet APF wouldn't handle that merging well anyway)

Reply:
Right now we have

```
package.json
lib/package.json
lib/src/..
```

Ideally I'd like to combine these into a single `package.json` and `src` dir. But if that is not possible that's fine